src/gmail.js

```diff
@@ -18,13 +18,8 @@ var Gmail = function(localJQuery) {
     } else if (typeof jQuery !== "undefined") {
         $ = jQuery;
     } else {
-        // try load jQuery through node.
-        try {
-            $ = require("jquery");
-        }
-        catch(err) {
-            // else leave $ undefined, which may be fine for some purposes.
-        }
+        // leave $ undefined, which may be fine for some purposes.
+        console.warn("GmailJS: jQuery is not provided in constructor parameters, DOM functions may not work.");
```

Comment:
I see what you're trying to do here, but I think we want to be even more explicit.

I think this early in the transition, opting out from jQuery should be something that requires extra effort, and ending up without jQuery is not something which can be done accidentally.

how about....

````javascript
var Gmail = function(localJQuery) {
  var $;
    if (typeof localJQuery !== "undefined") {
        $ = localJQuery;
    } else if (typeof jQuery !== "undefined") {
        $ = jQuery;
    } else if (typeof localJQuery === "boolean" && localJQuery === false} {
        // leave $ undefined, which may be fine for some purposes.
    } else {
        throw Error("GmailJS requires jQuery to be provided as a constructor argument!");
    }
}
````

Reply:
Makes sense. In that case, opting out from using jQuery would be really explicit, so we could even think of returning raw `HTMLElement`s instead of jQuery wrappers.

Reply:
Updated.